lib/go/thrift/simple_server.go

```diff
@@ -183,9 +186,6 @@ func (p *TSimpleServer) processRequests(client TTransport) error {
 			log.Printf("error processing request: %s", err)
 			return err
 		}
-		if err, ok := err.(TApplicationException); ok && err.TypeId() == UNKNOWN_METHOD {
-			continue
-		}
```

Comment:
I would doubt that. If that is true that would probably qualify for a bug report. I would not expect a formatting tool to arbitrarily delete valid code lines.